code/game/gamemodes/cult/runes.dm

```diff
@@ -38,11 +39,30 @@ To draw a rune, use an arcane tome.
 
 	var/req_keyword = 0 //If the rune requires a keyword - go figure amirite
 	var/keyword //The actual keyword for the rune
+	var/image/ai_image
```

Comment:
make this ```/static```, so they all reuse the same image, 1 image many runes.
(also check below, ```if(!ai_image) ... do image shit...```)